script/cfg.py

```diff
@@ -52,6 +52,14 @@ def rsync_commands(checksum):
         echo "rsync --timeout=3600 -tlp4 --specials PRODUCTISOPATH/${iso_folder[$flavor]}*$src.sha256 /var/lib/openqa/factory/other/$dest.sha256"'''
     return res
 
+def rsync_iso_rename_suse_microos(version):
+    if '5.0' in version:
```

Comment:
to make this valid in future releases, should we just check for `5.` ?
Actually we should also make sure that we don't hit false positives like `XYZ5.0` cases..
I found this method that might help
```
version.startswith('5.')
```

or with regex
```
import re
if re.match(r'^5.', version):
```

Reply:
Is this functionality for sle? Then maybe just define an empty abstract function here and overload it in the plugin, so we have all sle-specific functionality in one place? E.g. constant `openqa_call_legacy_builds` was added with similar purpose, it may be lambda or method as well.